docs/cmdlet-example/visual-studio-simple-example.md

```diff
@@ -0,0 +1,128 @@
+# Building a C# Cmdlet with Visual Studio
+
+This document describes steps for building a C# Cmdlet with Visual Studio in 2 ways:
+
+* [Building a C# Cmdlet for PowerShell Core](#building-a-c-cmdlet-for-powershell-core-with-visual-studio)
+* [Building a C# Cmdlet for PowerShell Standard 3.0](#building-a-c-cmdlet-for-powershell-standard-30-with-visual-studio)
+
+## Building a C# Cmdlet for PowerShell Core with Visual Studio
+
+This demonstrates how to build your own C# cmdlet for PowerShell Core with Visual Studio.
```

Comment:
I think it's worth explicitly saying something like:

Targeting for PowerShell Core means that the cmdlet may not work against Windows PowerShell if you take dependencies on new APIs introduced in PowerShell Core.

Reply:
Good point; updated.